src/components/scenes/EditTokenScene.tsx

```diff
@@ -202,13 +226,17 @@ function EditTokenSceneComponent(props: Props) {
             placeholder={translateDescription(item.displayName)}
             keyboardType={item.type === 'number' ? 'numeric' : 'default'}
             value={location.get(item.key) ?? ''}
-            onChangeText={value =>
+            onChangeText={value => {
               setLocation(location => {
                 const out = new Map(location)
                 out.set(item.key, value.replace(/\s/g, ''))
                 return out
               })
-            }
+
+              if (item.key === 'contractAddress') {
+                autoCompleteToken(value).catch(console.error)
```

Comment:
Ok, so in the other PR's, we talked about hiding errors in at the plugin level. We might even do that here too (`.catch(() => [])`), so we don't bother logging the errors to the console. It could be quite spammy if the internet is down.